hydra-node/json-schemas/api.yaml

```diff
@@ -758,6 +758,7 @@ components:
     Greetings:
       type: object
       required:
+        - tag
```

Comment:
Why did you have to introduce a tag here?

Reply:
our api.yaml already had:
```yaml
properties:
        tag:
          type: string
          enum: ["Greetings"]
```

I chose to make it required instead of removing it